web/src/components/dashboards/usePerses.ts

```diff
@@ -50,9 +47,9 @@ const reducer = (state: State, action: Action): State => {
   }
 };
 
-export const usePerses = () => {
-  const dashboardsAbort = useRef<() => void | undefined>();
+const URL_POLL_DEFAULT_DELAY = 15000; // 15 seconds
```

Comment:
I think a better solution would be to re fetch the dashboards only when opening the dropdown. Polling consumes additional resources, and in this case the list of dashboards is not constantly available but only when the dropdown opens